admin/developers_tool_kit.php

```diff
@@ -22,6 +22,7 @@
 $configuration_key_lookup = (isset($_POST['configuration_key'])) ? $_POST['configuration_key'] : '';
 $default_context_lines = isset($_POST['context_lines']) ? (int)$_POST['context_lines'] : $default_context_lines;
 $case_sensitive = (isset($_POST['case_sensitive']) && $_POST['case_sensitive']);
+$include_plugins = (isset($_POST['include_plugins']) && $_POST['include_plugins']);

```

Comment:
Why not simplify to `$include_plugins = !empty($_POST['include_plugins']);`?  Same on line 153.

Reply:
I suppose that consistency, even if not simplified, is good.